index.d.ts

```diff
@@ -213,6 +213,30 @@ export interface FastifyGQLCommonOptions {
    * receive an array of responses within a single request.
    */
   allowBatchedQueries?: boolean;
+
+  /**
+   * Settings for GraphQL Playground. These settings only apply if `graphiql` parameter is set to 'playground'.
+   * The most current GraphQL Playground code is loaded via CDN, so new configuration settings may be available.
+   * See https://github.com/prisma-labs/graphql-playground#usage for the most up-to-date list.
+   */
+  playgroundSettings?: {
```

Comment:
Since `graphql-playground` is not currently included as a dependency in this repo's `package.json`, I went ahead and just copied over the settings from the upstream repo. The other alternative is to include `graphql-playground-react` as a dependency for the types. It'd need to be a `dependency` vs. a `devDependency` if folks want to go that route.